<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1449</width>
    <height>555</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>30</y>
      <width>1171</width>
      <height>461</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 75 48pt &quot;MS Shell Dlg 2&quot;;
font: 8pt &quot;Segoe Fluent Icons&quot;;
color: rgb(170, 0, 0);
font: 36pt &quot;Kristen ITC&quot;;
font: 25 26pt &quot;Yu Gothic UI Light&quot;;
 border: none;
    background-color:          #fff7fa

 ;</string>
    </property>
    <property name="title">
     <string>       AssurePlus</string>
    </property>
    <widget class="QPushButton" name="pushButton_home">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>110</y>
       <width>71</width>
       <height>61</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true"> QPushButton
{
    border: none;
    background-color:          #ffeaea
 ;
    color: black;
      font-family: Tahoma, Verdana, sans-serif;
     font-size: 17px;
    border-radius: 50px;
}

QPushButton:hover
{
   
	background-color:       #e40b0b ;
 ;
    border: 2px solid black;
}

QPushButton:pressed
{
    background-color:   #f3cbcb ;
    border: 2px solid white;
}</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_3">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>170</y>
       <width>71</width>
       <height>51</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">  QPushButton
{
    border: none;
    background-color:          #ffeaea
;
    color: black;
      font-family: Tahoma, Verdana, sans-serif;
     font-size: 17px;
    border-radius: 50px;
}

QPushButton:hover
{
   
	background-color:       #e40b0b ;
 ;
    border: 2px solid black;
}

QPushButton:pressed
{
    background-color:   #f3cbcb ;
    border: 2px solid white;
}</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>220</y>
       <width>71</width>
       <height>51</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true"> QPushButton
{
    border: none;
    background-color:          #ffeaea
 ;
    color: black;
      font-family: Tahoma, Verdana, sans-serif;
     font-size: 17px;
    border-radius: 50px;
}

QPushButton:hover
{
   
	background-color:       #e40b0b ;
 ;
    border: 2px solid black;
}

QPushButton:pressed
{
    background-color:   #f3cbcb ;
    border: 2px solid white;
}</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_4">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>270</y>
       <width>71</width>
       <height>61</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">  QPushButton
{
    border: none;
    background-color:         #ffeaea
;
    color: black;
      font-family: Tahoma, Verdana, sans-serif;
     font-size: 17px;
    border-radius: 50px;
}

QPushButton:hover
{
   
	background-color:       #e40b0b ;
 ;
    border: 2px solid black;
}

QPushButton:pressed
{
    background-color:   #f3cbcb ;
    border: 2px solid white;
}</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>330</y>
       <width>71</width>
       <height>61</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">  QPushButton
{
    border: none;
    background-color:        #ffeaea
;
    color: black;
      font-family: Tahoma, Verdana, sans-serif;
     font-size: 17px;
    border-radius: 50px;
}

QPushButton:hover
{
   
	background-color:       #e40b0b ;
 ;
    border: 2px solid black;
}

QPushButton:pressed
{
    background-color:   #f3cbcb ;
    border: 2px solid white;
}</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_5">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>390</y>
       <width>71</width>
       <height>61</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true"> 
QPushButton {
    border: none;
    background-color: #e40b0b;  
    color: white;   
    font-family: Tahoma, Verdana, sans-serif;
    font-size: 17px;
    border-radius: 50px;
}

QPushButton:hover {
    background-color: #ffeaea; 
    border: 2px solid black;
}

QPushButton:pressed {
    background-color: #e40b0b; 
    border: 2px solid white;
}

</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>120</y>
       <width>51</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">image: url(:/res/home.png);</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>180</y>
       <width>51</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">image: url(:/res/trousse-de-premiers-secours.png);</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="label_9">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>280</y>
       <width>51</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">image: url(:/res/signe.png);</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="label_11">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>230</y>
       <width>51</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">image: url(:/res/client.png);</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="label_12">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>340</y>
       <width>51</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">image: url(:/res/employe-de-bureau.png);</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="label_13">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>400</y>
       <width>51</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">image: url(:/res/editer.png);</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="label_15">
     <property name="geometry">
      <rect>
       <x>-10</x>
       <y>-20</y>
       <width>101</width>
       <height>101</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">image: url(:/res/logo.png);
font: 8pt &quot;MS Shell Dlg 2&quot;;</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="label_17">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>130</y>
       <width>251</width>
       <height>111</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(0, 0, 0);
font: 75 17pt  ;
  font: bold italic large serif;</string>
     </property>
     <property name="text">
      <string>Date</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_18">
     <property name="geometry">
      <rect>
       <x>500</x>
       <y>10</y>
       <width>251</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">  font: Tahoma, Verdana, sans-serif;
color:black;
font-size :20px;
  color: black;
</string>
     </property>
     <property name="text">
      <string>Gestion des r√©clamations</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_20">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>310</y>
       <width>251</width>
       <height>111</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(0, 0, 0);
font: 75 17pt  ;
  font: bold italic large serif;</string>
     </property>
     <property name="text">
      <string>Description</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_Description">
     <property name="geometry">
      <rect>
       <x>340</x>
       <y>350</y>
       <width>321</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 255);
font: 9pt &quot;Arial&quot;;</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_ajouter">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>420</y>
       <width>141</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true"> QPushButton
{
    border: none;
    background-color:          #ffeaea
 ;
    color: black;
      font-family: Tahoma, Verdana, sans-serif;
     font-size: 17px;
    border-radius: 50px;
}

QPushButton:hover
{
   
	background-color:       #e40b0b ;
 ;
    border: 2px solid black;
}

QPushButton:pressed
{
    background-color:   #f3cbcb ;
    border: 2px solid white;
}</string>
     </property>
     <property name="text">
      <string>Ajouter</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_supprimer">
     <property name="geometry">
      <rect>
       <x>510</x>
       <y>420</y>
       <width>141</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true"> QPushButton
{
    border: none;
    background-color:          #ffeaea
 ;
    color: black;
      font-family: Tahoma, Verdana, sans-serif;
     font-size: 17px;
    border-radius: 50px;
}

QPushButton:hover
{
   
	background-color:       #e40b0b ;
 ;
    border: 2px solid black;
}

QPushButton:pressed
{
    background-color:   #f3cbcb ;
    border: 2px solid white;
}</string>
     </property>
     <property name="text">
      <string>Supprimer</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_16">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>170</y>
       <width>41</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">image: url(:/res/date.png);</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="label_19">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>350</y>
       <width>61</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">image: url(:/res/description.png);</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_modifier">
     <property name="geometry">
      <rect>
       <x>1020</x>
       <y>420</y>
       <width>141</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true"> QPushButton
{
    border: none;
    background-color:          #ffeaea
 ;
    color: black;
      font-family: Tahoma, Verdana, sans-serif;
     font-size: 17px;
    border-radius: 50px;
}

QPushButton:hover
{
   
	background-color:       #e40b0b ;
 ;
    border: 2px solid black;
}

QPushButton:pressed
{
    background-color:   #f3cbcb ;
    border: 2px solid white;
}</string>
     </property>
     <property name="text">
      <string>Modifier</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_stat">
     <property name="geometry">
      <rect>
       <x>500</x>
       <y>80</y>
       <width>141</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true"> QPushButton
{
    border: none;
    background-color:          #ffeaea
 ;
    color: black;
      font-family: Tahoma, Verdana, sans-serif;
     font-size: 17px;
    border-radius: 50px;
}

QPushButton:hover
{
   
	background-color:       #e40b0b ;
 ;
    border: 2px solid black;
}

QPushButton:pressed
{
    background-color:   #f3cbcb ;
    border: 2px solid white;
}</string>
     </property>
     <property name="text">
      <string>Statistique</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_list">
     <property name="geometry">
      <rect>
       <x>670</x>
       <y>80</y>
       <width>181</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true"> QPushButton
{
    border: none;
    background-color:          #ffeaea
 ;
    color: black;
      font-family: Tahoma, Verdana, sans-serif;
     font-size: 17px;
    border-radius: 50px;
}

QPushButton:hover
{
   
	background-color:       #e40b0b ;
 ;
    border: 2px solid black;
}

QPushButton:pressed
{
    background-color:   #f3cbcb ;
    border: 2px solid white;
}</string>
     </property>
     <property name="text">
      <string>Liste des reclamations</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_21">
     <property name="geometry">
      <rect>
       <x>440</x>
       <y>10</y>
       <width>41</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">image: url(:/res/editer.png);</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QDateEdit" name="DateEdit">
     <property name="geometry">
      <rect>
       <x>330</x>
       <y>160</y>
       <width>281</width>
       <height>41</height>
      </rect>
     </property>
     <property name="dateTime">
      <datetime>
       <hour>0</hour>
       <minute>0</minute>
       <second>0</second>
       <year>2020</year>
       <month>12</month>
       <day>30</day>
      </datetime>
     </property>
     <property name="minimumDateTime">
      <datetime>
       <hour>0</hour>
       <minute>0</minute>
       <second>0</second>
       <year>2020</year>
       <month>1</month>
       <day>1</day>
      </datetime>
     </property>
    </widget>
    <widget class="QTableView" name="tableView">
     <property name="geometry">
      <rect>
       <x>670</x>
       <y>120</y>
       <width>421</width>
       <height>281</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 255);
font: italic 8pt &quot;Arial&quot;;</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_trier">
     <property name="geometry">
      <rect>
       <x>360</x>
       <y>80</y>
       <width>141</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true"> QPushButton
{
    border: none;
    background-color:          #ffeaea
 ;
    color: black;
      font-family: Tahoma, Verdana, sans-serif;
     font-size: 17px;
    border-radius: 50px;
}

QPushButton:hover
{
   
	background-color:       #e40b0b ;
 ;
    border: 2px solid black;
}

QPushButton:pressed
{
    background-color:   #f3cbcb ;
    border: 2px solid white;
}</string>
     </property>
     <property name="text">
      <string>Trier</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_export">
     <property name="geometry">
      <rect>
       <x>220</x>
       <y>80</y>
       <width>141</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true"> QPushButton
{
    border: none;
    background-color:          #ffeaea
 ;
    color: black;
      font-family: Tahoma, Verdana, sans-serif;
     font-size: 17px;
    border-radius: 50px;
}

QPushButton:hover
{
   
	background-color:       #e40b0b ;
 ;
    border: 2px solid black;
}

QPushButton:pressed
{
    background-color:   #f3cbcb ;
    border: 2px solid white;
}</string>
     </property>
     <property name="text">
      <string>Export </string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1449</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
